robottelo/constants/__init__.py

```diff
@@ -1104,6 +1113,12 @@ class Colored(Box):
         'edit_operatingsystems',
         'destroy_operatingsystems',
     ],
+    'OrcharhinoCore': [
```

Comment:
I'm not sure about these 🐋 Orcharhino 🦏 parts. I'm not sure they should be present in this repo. I also don't think it's my decision. What do others think? @SatelliteQE/robottelo-tier-1-reviewers @SatelliteQE/robottelo-tier-2-reviewers @lvrtelov ?

Reply:
I neither agree or disagree with you.
But where does it end? Will we have all potential Foreman forks' code here in this repo?

Reply:
:D would that be a bad thing ;)

Can we maybe find a solution to (additionally) define it in the settings?

Reply:
@jyejare If you are asking me whether Satellite supports Orcharhino... then no, no it doesn't

Reply:
@dosas "We" can definitely find a solution but I'm afraid I'm not the person who should decide this. It's a management question, not a technical one.

Reply:
sounds like it. In that case I will remove the orcharhino part and maintain it downstream.

Reply:
@dosas @lhellebr while this repo did start out as more foreman-focused, it grew/matured almost entirely downstream/Satellite environment. Due to this, there is quite a bit of Satellite-specific language in the codebase. Is this something that could be changed, likely yes. Is this change something we're likely to prioritize in the near-term, probably not.
However, that doesn't mean that this repo's design can't be re-evaluated when changes like this might expose a design limitation.